cfme/common/provider.py

```diff
@@ -756,7 +756,7 @@ def _fill_credential(form, cred, validate=None):
                 'token_verify_secret': cred.verify_token,
                 'validate_btn': validate
             })
-            if validate:
+            if validate_hawkular:
```

Comment:
@izapolsk Per Jan's request I included the changes in the test that will use this path:
https://github.com/ManageIQ/integration_tests/pull/4405